prow/proxy-common.inc

```diff
@@ -50,14 +50,18 @@ export BAZEL_OUT
 # Disable RBE execution until we have LARGE_MACHINE instances provisioned for building V8.
 export BAZEL_BUILD_RBE_JOBS="${BAZEL_BUILD_RBE_JOBS:-0}"
```

Comment:
nit- the comment here is outdated, the new issue is the tool chain used in RBE is too new for us. We will need to see if we can switch it over to our image